encrpt/aes_gcm_test.go

```diff
@@ -2,18 +2,13 @@ package encrpt
 
 import (
 	"testing"
-	"github/wziww/medusa/config"
+	// "github/wziww/medusa/config"
```

Comment:
Useless code should be removed

Reply:
https://github.com/xiaoxiaoking/medusa/commit/de1a3fdfdb9877d2c8de4b6c7e14628db240b8ec